vnext/Folly/Folly.vcxproj

```diff
@@ -72,7 +72,7 @@
       <ObjectFileName>$(IntDir)\portabilityString.obj</ObjectFileName>
     </ClCompile>
     <ClCompile Include="$(FollyDir)\folly\Demangle.cpp" />
-    <ClCompile Include="$(FollyDir)\folly\dynamic.cpp" />
+    <ClCompile Include="$(FollyDir)\folly\json\dynamic.cpp" />
     <ClCompile Include="$(FollyDir)\folly\json.cpp" />
```

Comment:
IIRC didn't the json.cpp file move too? We should make sure all of the files (esp the ones we have overrides for) get updated.

Reply:
fixed and went over the files in this PR that changed: https://github.com/facebook/folly/commit/cb479681ca490948d9b83a1ccff703e61c443fc4#diff-bab11a892f5cba24e9ec1bf349c6d7fb7e5b5732aa3ea2125f66b47bc9b6cc5c. Worked before because the old dependencies still link to the new files